homework/english/raby_alexandr/lection1/vocabulary.md

```diff
@@ -0,0 +1,17 @@
+1. I boot Win10 before work.
+2. I create my first homework in Github.
```

Comment:
Created

Reply:
We study the Present Simple and Present Continuous. Why use the Past Simple?

Reply:
So maybe u wanted to say  "I am creating"? See the rules more attentively